src/middleware/middleware.js

```diff
@@ -31,3 +37,42 @@ export const validateUserInput = (req, res, next) => {
   }
   return next();
 };
+
+export const validatePropertyInput = (req, res, next) => {
+  const {
+    image, title, address, landArea, noOfRoom, noOfBath,
+    noOfGarage, noOfStore, yearBuild, purpose
+  } = req.body;
+
+  if (!checkImageExtension(image)) {
+    res.status(400).send('Property image must be uploaded');
+  }
+  if (title === '') {
```

Comment:
You need to revisit this logic, does that mean if I send empty space "    " it will pass? What you want to do instead is to trim it then check the length. Example:

if(!title || title.trim().length > 0){

// whatever

}

Reply:
Same thing goes for other validation logic, you check an example we made for the signup validation.